lit_gpt/lora.py

```diff
@@ -354,36 +335,17 @@ def zero_pad(self, x: torch.Tensor) -> torch.Tensor:
         result = result.index_copy(1, self.lora_ind, x.reshape(-1, shape))  # (4096, 256)
         return result.view((*x.shape[:-1], self.out_features)).transpose(0, 1)  # (64, 64, 384)
 
-    def train(self, mode: bool = True):
```

Comment:
The previous code supported "unmerging" too, but we don't need it. If you want to keep it nonetheless (e.g. in a unmerge method), let me know.